docs/book/src/topics/capibmadm/vpc/key.md

```diff
@@ -0,0 +1,20 @@
+## PowerVS VPC Commands
```

Comment:
```suggestion
## VPC SSH key Commands
```

Reply:
@pkenchap can you please also fix same line in image doc file.

Reply:
done